sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala

```diff
@@ -30,12 +30,13 @@ class DataSourceScanExecRedactionSuite extends QueryTest with SharedSQLContext {
 
   override protected def sparkConf: SparkConf = super.sparkConf
     .set("spark.redaction.string.regex", "file:/[\\w_]+")
+    .set(SQLConf.USE_V1_SOURCE_READER_LIST.key, "orc")
```

Comment:
why this change?

Reply:
Otherwise, we still need to set the conf to use Parquet V1 after Parquet V2 is migrated.